src/app/vault/ciphers.component.html

```diff
@@ -39,8 +39,8 @@
     </ng-container>
 </div>
 <div class="footer">
-    <button appBlurClick (click)="addCipher()" (contextmenu)="addCipherOptions()" class="block primary"
-        appA11yTitle="{{'addItem' | i18n}}">
+    <button appBlurClick (click)="(deleted ? false : addCipher())" (contextmenu)="addCipherOptions()" 
```

Comment:
Is the `(click)` event still fired when disabled?

Reply:
No, I'll remove that... was my bad my [disabled] prop wasn't working, but I just wasn't patient enough for electron to rebuild.

Reply:
Corrected.